File: tools/dbench/tasks/f2_adults.py
names = []
with open("users.txt") as f:
    for line in f:
        parts = line.split()
        if len(parts) != 3:
            continue
        try:
            age = int(parts[2])
        except ValueError:
            continue
        if age > 25:
            names.append(parts[1])
print(" ".join(names))

File: tools/dbench/answers/rust_sonnet/s1/f2_adults.rs
use std::fs;

fn main() {
    let contents = fs::read_to_string("users.txt").unwrap_or_default();
    let mut names = Vec::new();
    for line in contents.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i64>() {
                if age > 25 {
                    names.push(parts[1].to_string());
                }
            }
        }
    }
    println!("{}", names.join(" "));
}
